packages/ckeditor5-engine/src/model/documentselection.js

```diff
@@ -882,6 +917,54 @@ class LiveSelection extends Selection {
 		}
 	}
 
+	_updateMarker( marker, markerRange ) {
+		if ( !this._observedMarkerGroups.length ) {
```

Comment:
Not needed - this is covered by the next if pretty well.

Reply:
This was to make it return as early as possible. There is no point in splitting marker name and looking for it in the Set if it is empty

Reply:
The gain here is very minimal. In such situations, I prefer fewer lines of code. I guess it is a bit of a personal preference. But really, you don't gain anything here by leaving early.

Reply:
ok

Reply:
done